libs/domain/picking/src/services/picking-list-default.service.ts

```diff
@@ -16,8 +16,9 @@ export class PickingListDefaultService implements PickingListService {
   startPicking(pickingList: PickingList): Observable<PickingList | null> {
     this.upcomingPickingListId$.next(pickingList.id);
     return this.adapter.startPicking(pickingList).pipe(
-      catchError(() => {
+      catchError((e) => {
         this.upcomingPickingListId$.next(null);
+        throw e;
         return of(null);
```

Comment:
You should not throw and return as both do not makes sense together.
Instead you can return an error stream using `throwError(() => e)` operator.